nodejs/week2/app.js

```diff
@@ -0,0 +1,93 @@
+import express from "express";
+import fs from "fs";
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+//Support parsing JSON requests
+app.use(express.json());
+
+let documents;
+try {
+  const data = fs.readFileSync("documents.json");
+  documents = JSON.parse(data);
+} catch (error) {
+  console.error("Error reading documents.json:", error);
+  documents = [];
+}
```

Comment:
Put the above into a function, is is against best practice to have code executed in the root scope of a backend. hope this makes sense :)
you can then do:
```
const documents = functionThatReadsTheFile()
```